pyodide-build/README.md

```diff
@@ -5,6 +5,14 @@ Tools for building Pyodide.
 See [http://github.com/pyodide/pyodide](http://github.com/pyodide/pyodide) for
 more information.
 
+## Upgrading pyodide to a new version of CPython
+
+For example: `v3.11.1` -> `v3.11.2`
+
+1. Download the **Gzipped source tarball** https://www.python.org/downloads/release/python-3112/ into `downloads`
+2. `shasum -a 256 downloads/Python-3.11.2.tgz > cpython/checksums`
+3. `git grep --name-only "3.11.1"` # These files should be updated
```

Comment:
```suggestion
1. Download the **Gzipped source tarball** at https://www.python.org/downloads/release/python-3112 into `downloads`
2. `shasum -a 256 downloads/Python-3.11.2.tgz > cpython/checksums`
3. `git grep --name-only "3.11.1" ` # These files should be updated.
4. Create a new image from the updated `Dockerfile`
5. Upload the new image to https://hub.docker.com/r/pyodide/pyodide-env
6. Modify the image name in `.circleci/config.yml` to match the name Docker Hub.
7. Rebase any patches which do not apply cleanly.
8. Create a pull request and fix any failing tests.  This may be complicated for major releases.
```